ui/apps/platform/src/Containers/Administration/Events/AdministrationEventsTable.css

```diff
@@ -1,7 +1,8 @@
-#AdministrationEventsTable td:first-child {
-    width: 14px;
-}
-
-#AdministrationEventsTable td:first-child[data-label="Level icon"] {
-    vertical-align: middle;
+/*
+ * With sort button, count column head needs more than pf-u-text-align-right utility class
+ * to align at right for numeric data.
+ */
+#AdministrationEventsTable th:last-child button.pf-c-table__button {
+    margin-left: auto;
+    padding-right: 0;
```

Comment:
I thought we had a reusable class for right-justified table headers somewhere?

Reply:
Aha, good point. I will search.

Most of the tables align numeric values left, although right is a common convention.

Reply:
Apparently not: `margin-left: auto` has no occurrences

Closest was **vertical** alignment of sort indicator:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/css/trumps.css#L138-L141

Reply:
It may be done with PF classes or Flex components, but this is fine for now.